### PR TITLE
Thunder recommendation on db

### LINKED ISSUE #963: Save/retrieve Ercole recommendation to/from DB
Move currently ercole recommendations management into a specific job and store the retrieved information in a DB collection.
Then convert the APIs into a single API to retrieve all the ercole recommendations.

### PR DESCRIPTION
**Pull request by anbrenna**
Fix #963

### CODE REVIEW

File: thunder-service/database/oci_recommendations.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+func (md *MongoDatabase) GetLastOciSeqValue() (uint64, error) {
+	ctx := context.TODO()
+
+	findOptions := options.Find()
+	findOptions.SetSort(bson.M{"seqValue": -1})
+
+	cur, err := md.Client.Database(md.Config.Mongodb.DBName).Collection(OciRecommendation_collection).Find(ctx, bson.D{}, findOptions)
+	if err != nil {
+		return 999999, utils.NewError(err, "DB ERROR")
```
Comment (cbiglioli) [inline thunder-service/database/oci_recommendations.go:143]: Could you try to change 999999 with math.MaxUint64?

File: thunder-service/database/oci_recommendations.go [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+	}
+
+	ociRecommendations := make([]model.OciRecommendation, 0)
+	if err := cur.All(context.TODO(), &ociRecommendations); err != nil {
+		return 999999, utils.NewError(err, "DB ERROR")
+	}
+
+	if err := cur.Err(); err != nil {
+		return 999999, utils.NewError(err, "DB ERROR")
```
Comment (cbiglioli) [inline thunder-service/database/oci_recommendations.go:152]: Could you try to change 999999 with math.MaxUint64?

File: thunder-service/database/oci_recommendations.go [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+	findOptions.SetSort(bson.M{"seqValue": -1})
+
+	cur, err := md.Client.Database(md.Config.Mongodb.DBName).Collection(OciRecommendation_collection).Find(ctx, bson.D{}, findOptions)
+	if err != nil {
+		return 999999, utils.NewError(err, "DB ERROR")
+	}
+
+	ociRecommendations := make([]model.OciRecommendation, 0)
+	if err := cur.All(context.TODO(), &ociRecommendations); err != nil {
+		return 999999, utils.NewError(err, "DB ERROR")
```
Comment (cbiglioli) [inline thunder-service/database/oci_recommendations.go:148]: Could you try to change 999999 with math.MaxUint64?

File: thunder-service/database/oci_recommendations.go (hunk 1 above)
Comment (anbrenna) [reply thunder-service/database/oci_recommendations.go]: Fixed

File: thunder-service/database/oci_recommendations.go (hunk 3 above)
Comment (anbrenna) [reply thunder-service/database/oci_recommendations.go]: Fixed

**Review by cbiglioli — APPROVED** [review]

**Pull request merged**